src/BloomBrowserUI/bookEdit/readerSetup/ReaderSetup.jade

```diff
@@ -1,189 +0,0 @@
-doctype html
```

Comment:
You did several merges, maybe one of them did it? If it shouldn't go away please put it back!